zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -85,48 +88,42 @@ class MeasurementType(enum.IntFlag):
             attr=ElectricalMeasurement.AttributeDefs.rms_current_ph_c.name,
             config=REPORT_CONFIG_OP,
         ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_b.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_c.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
         AttrReportConfig(
             attr=ElectricalMeasurement.AttributeDefs.rms_voltage.name,
             config=REPORT_CONFIG_OP,
         ),
         AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_voltage_max.name,
-            config=REPORT_CONFIG_DEFAULT,
+            attr=ElectricalMeasurement.AttributeDefs.rms_voltage_ph_b.name,
+            config=REPORT_CONFIG_OP,
         ),
         AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.ac_frequency.name,
+            attr=ElectricalMeasurement.AttributeDefs.rms_voltage_ph_c.name,
             config=REPORT_CONFIG_OP,
         ),
         AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.ac_frequency_max.name,
-            config=REPORT_CONFIG_DEFAULT,
+            attr=ElectricalMeasurement.AttributeDefs.ac_frequency.name,
+            config=REPORT_CONFIG_OP,
         ),
     )
     ZCL_INIT_ATTRS = {
         ElectricalMeasurement.AttributeDefs.ac_current_divisor.name: True,
         ElectricalMeasurement.AttributeDefs.ac_current_multiplier.name: True,
+        ElectricalMeasurement.AttributeDefs.ac_frequency_divisor.name: True,
```

Comment:
sorted this dict alphabetically to make it easier to look things up. let me know if that is not OK and I'll revert to the previous order